dev/io.openliberty.mail.2.1.internal_fat/build.gradle

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+/* Define greenmail dependencies */
+dependencies {
+  requiredLibs project(':com.ibm.ws.org.slf4j.api'),
+               project(':com.ibm.ws.org.slf4j.jdk14'),
+               'com.ibm.ws.com.icegreen:greenmail:2.0.0.fd826131f671b0d3230b0d0e1638d8c4',
+               // Deps needed for when running on Java 9 (EE-type APIs were removed from JDK)
+               'com.sun.mail:jakarta.mail:2.0.0',
```

Comment:
we should change this one to the new IR (angus mail's) coordinates:  org.eclipse.angus:angus-mail:1.0.0